tests/fixtures/sourcemaps/01_sourcemap_expansion/test.min.js

```diff
@@ -0,0 +1,2 @@
+var makeAFailure=function(){function n(n){}function e(n){throw new Error("failed!")}function r(r){var i=null;if(r.failed){i=e}else{i=n}i(r)}function i(){var n={failed:true,value:42};r(n)}return i}();
+//# sourceMappingURL=test.min.js.map
```

Comment:
Hm, I’m actually surprised this works. Does it really allow spaces in between `//#sourceMappingURL`?

Reply:
It's mandatory :) - https://github.com/getsentry/rust-sourcemap/blob/1ba39818c2e28e8c46d284ee110df86680481584/src/detector.rs#L76